src/ui/ResponsiveComponents/ResponsiveComponentsManager.ts

```diff
@@ -121,6 +128,13 @@ export class ResponsiveComponentsManager {
       if (this.isFacet(ID)) {
```

Comment:
Instead of keeping a reference on every component we want to call `registerComponent` on, we could add this method on the interface `ResponsiveComponent` and loop on `this.responsiveComponents` and call it if it's there. It would be like a notification that a component of the same type has been registered.